@stellar/design-system-website/src/components/Details/index.tsx

```diff
@@ -55,6 +64,102 @@ export const Details = () => {
     ));
   };
 
+  const renderPropsTable = (
+    renderProps: ComponentProp[],
+    renderExternalProps?: ComponentExternalProps,
+  ) => (
+    <table>
+      <thead>
+        <tr>
+          <th>Prop</th>
+          <th>Type</th>
+          <th>Default</th>
+          <th>Optional</th>
+          <th>Description</th>
+        </tr>
+      </thead>
+      <tbody>
+        {renderProps.map((componentProp) => (
+          <tr key={`${componentProp.prop}-${Math.random()}`}>
```

Comment:
Best not to use `Math.random()` in React keys. It would probably be better to use array index and disable the linting warning for it:

https://reactjs.org/docs/reconciliation.html#tradeoffs